lib/expand/join.go

```diff
@@ -1286,6 +1292,10 @@ func isExpandOperationReady(state string) (bool, error) {
 	}
 }
 
+func thisPeerPartOfInstallState(addr string, servers []storage.Server) bool {
```

Comment:
Nit: I think you could just accept `storage.Servers` and Go would do the type conversion automatically and you won't have to do it below.